cmd/ocagent/main.go

```diff
@@ -80,19 +79,22 @@ func main() {
 }
 
 func runOCAgent() {
-	yamlBlob, err := ioutil.ReadFile(configYAMLFile)
+	viperCfg.SetConfigFile(configYAMLFile)
```

Comment:
Where did `viperCfg` come from? Or where is it declared?

Reply:
line 58

Reply:
Cool, thanks!